taskcluster/tc-py-utils.sh

```diff
@@ -286,17 +286,17 @@ get_python_pkg_url()
 get_tflite_python_pkg_name()
 {
   # Default to deepspeech package
-  local _pkgname="deepspeech_tflite"
+  local _pkgname="mozilla_voice_stt_tflite"
```

Comment:
so here we have underscore where elsewhere you set dash :)

Reply:
Huh. Looks like Python/pip normalizes dashes to underscores anyway.